projects/components/src/filtering/filter/filter-operators.ts

```diff
@@ -44,6 +46,8 @@ export const toUrlFilterOperator = (operator: FilterOperator): UrlFilterOperator
       return UrlFilterOperator.In;
     case FilterOperator.ContainsKey:
       return UrlFilterOperator.ContainsKey;
+    case FilterOperator.NotIn:
+      return UrlFilterOperator.ContainsKey;
```

Comment:
bad map

Reply:
yeah. Do you know we need three operator types? GqlOperators, FilterOperator and URLFilterOperator. Imo We can combine FilterOperator and URLFilterOperator and only use one.

Reply:
There's three different string representations of a user - the server's value (e.g. `EQUALS`), the value we display in the UI (`=`) and the value we use in a URL string where we can't use all specials chars (`_eq_`). We probably could try to consolidate such as by url encoding or similar, but seemed easy enough to maintain initially.